ocaml/testsuite/tests/typing-immediate/immediate.ml

```diff
@@ -37,14 +37,14 @@ end;;
 [%%expect{|
 module A :
   sig
-    type t [@@immediate]
-    type s = t [@@immediate]
+    type t : immediate
+    type s = t
```

Comment:
This has changed where annotations are printed; is that okay?

Reply:
See `Note [When to print layout annotations]` in Printtyp.